pkgs/development/python-modules/backports_shutil_get_terminal_size/default.nix

```diff
@@ -23,4 +31,4 @@ if !(pythonOlder "3.3") then null else buildPythonPackage rec {
     homepage = https://github.com/chrippa/backports.shutil_get_terminal_size;
     license = with lib.licenses; [ mit ];
   };
```

Comment:
Do you want to maintain this? Packages not fetched from PyPI need someone updating them by hand.

Reply:
I doubt there will be another update before the EOL of python 2.7. But sure, I'll maintain just in case.